tools/docker/cpu_tests.Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN git clone https://github.com/abhinavsingh/setup-bazel.git
 RUN bash ./setup-bazel/setup-bazel.sh 1.1.0
 
 COPY requirements.txt ./
-RUN pip install -r requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
This is not really necessary since we don't care about a few dozens of MB more  in image size. We're not going to push it/pull it.

Reply:
Thanks @gabrieldemarmiesse. This part is about the built image `layer`:  if the `requirements.txt` modified, the image will not be an updated one since there is cache here. Only if we are sure the file won't be changed. May I know more from your suggestions, @gabrieldemarmiesse?

Reply:
The pip cache is not present if the docker cache is invalidated. You can try and see for yourself :)

Reply:
Oh, yes, if you assigned the version number, it'll be updated. But, for `pathlib` and `pyyaml`, in `doc_requirement.txt`,  do we also assign version number?

Reply:
Pathlib is actually not needed in the requirements since it's in the standard library since py3. For pyyaml, it might be worth pinning but we don't get many benefits since it's a very stable library.

Reply:
Pinning is more for libraries where we might expect things to change frequently.

Reply:
Thanks, @gabrieldemarmiesse. I'll remove `pathlib`. Thanks a lot again for the comments!